lib/index.js

```diff
@@ -145,7 +145,7 @@ export class Baker extends EventEmitter {
       output: this.output,
       pathPrefix: this.pathPrefix,
       staticRoot: this.staticRoot,
-      basePath: this.basePath
+      basePath: this.basePath,
```

Comment:
remove trailing comma

Reply:
sadly this one is automatic. their pre-commit linter adds these trailing commas